docs/developer/app-store/apps/taxes/tax-codes.mdx

```diff
@@ -7,3 +7,25 @@ To illustrate the feature, imagine you are running a book store that sells comic
 You first create the "_Comic books_" Saleor tax class (_Configuration_ → _Taxes_ → _Tax classes_ in the Dashboard) and assign it to comic book products or their variants.
 
 In the _Tax code matcher_ section of the Taxes App, you will find your "Comicbook" tax class. You browse the tax codes of your tax provider and assign the correct one to the Saleor tax class. In AvaTax, that tax code may be "PB100817 - Comicbooks". The selected tax code will appear on the product line in AvaTax when the transaction is created.
+
+This feature can be used to also cover additional fees for an order. In the example below, we explain how to cope with one of them: [Colorado Retail Delivery Fee](https://tax.colorado.gov/retail-delivery-fee).
+
+## Example: Colorado Retail Delivery Fee in AvaTax
+
+[Colorado Retail Delivery Fee](https://tax.colorado.gov/retail-delivery-fee) is a fee charged by the state of Colorado for the delivery of retail goods. Let's explore how we can make sure the fee is included in the tax calculation done by AvaTax.
+
+1. Create a new tax class in Saleor and name it "_Colorado Retail Delivery Fee_".
+
+Go to _Configuration_ → _Taxes_ → _Tax classes_ in the Dashboard and click _Create class_. Name the tax class "_Colorado Retail Delivery Fee_", don't set any tax rates and click _Save_.
+
+2. Create a product in Saleor and assign the "_Colorado Retail Delivery Fee_" tax class to it.
```

Comment:
- Creating product type is probably required too
- We can suggest to set the product to be hidden in the product lists:
<img width="616" alt="image" src="https://github.com/saleor/saleor-docs/assets/5188636/386532f0-ce58-41f3-9543-13813a555b46">
- is there a way to prevent users from adding it to the cart by hand?

Reply:
I assume it's a free item with unlimited stock?

Reply:
@patrys Yes, that would be ideal. How would I go about creating such an item?

Reply:
You'd need to disable stock tracking and set the price to 0.

Reply:
updated @patrys